### PR TITLE
pipeline optimizations:  tweak vCPU and memory requirements

### PR DESCRIPTION
**Pull request by boris-dimitrov**

### CODE REVIEW

File: app/models/pipeline_run_stage.rb [hunk 1]
```diff
@@ -25,10 +23,19 @@ def install_pipeline
...
+      vcpus = Sample::DEFAULT_VCPUS
+      queue = Sample::DEFAULT_QUEUE
+    else
+      vcpus = Sample::DEFAULT_VCPUS_HIMEM
+      queue = Sample::DEFAULT_QUEUE_HIMEM
+    end
+    if pipeline_run.sample.job_queue.present?
+      queue = pipeline_run.sample.job_queue
+    end
+    command += " --storage /mnt=#{Sample::DEFAULT_STORAGE_IN_GB} --ecr-image idseq --memory #{memory} --queue #{queue} --vcpus #{vcpus}"
```
Comment (boris-dimitrov) [inline app/models/pipeline_run_stage.rb]: I think the latest changes cleaned this up a bit better.   Does it still look good, @cdebourcy?

**Comment by boris-dimitrov** [comment]
Testing new `idseq` queue in https://alpha.idseq.net/samples/321

**Comment by boris-dimitrov** [comment]
This passes tests but I am presently struggling to run it in localhost env (because my env is broken).

File: app/models/pipeline_run_stage.rb (hunk 1 above)
Comment (cdebourcy) [reply app/models/pipeline_run_stage.rb]: It looks good to me!

**Comment by boris-dimitrov** [comment]
As soon as this is deployed on production, we need to `update host_genomes SET job_queue=NULL`.

**Pull request merged**